anta/tests/routing/bgp.py

```diff
@@ -746,8 +751,17 @@ def test(self) -> None:
                 failures = deep_update(failures, failure)
                 continue
 
-            # Check each capability
+            # Fetching the capabilities output
             bgp_output = get_value(bgp_output, "neighborCapabilities.multiprotocolCaps")
+
+            # Verifies the only mentioned capabilities should be listed in case needed.
```

Comment:
i think we should create some helper functions and if strict is true then verification should done separately(only mentioned  capabilities should present  and it should not look for others. If strict is false then as the exiting code there we can converted into a helper function.

Reply:
Added helper function for the same.